diff --git a/src/llvm_core.cpp b/src/llvm_core.cpp
index ef2c6ad..f2b1f92 100644
--- a/src/llvm_core.cpp
+++ b/src/llvm_core.cpp
@@ -169,7 +169,7 @@ bool jitc_llvm_init() {
         jitc_llvm_shutdown();
     }

-    if (jitc_llvm_api_has_orcv2() && jitc_llvm_orcv2_init()) {
+    if (false && jitc_llvm_api_has_orcv2() && jitc_llvm_orcv2_init()) {
         jitc_llvm_use_orcv2 = true;
     } else if (jitc_llvm_api_has_mcjit() && jitc_llvm_mcjit_init()) {
         jitc_llvm_use_orcv2 = false;
